fields/types/html/HtmlField.js

```diff
@@ -1,5 +1,20 @@
-import { BlockMapBuilder, Editor, EditorState, KeyBindingUtil, Modifier, RichUtils, convertFromHTML, convertFromRaw, convertToRaw, getDefaultKeyBinding } from '@twreporter/draft-js';
```

Comment:
Should we try to standardize the rules we would like to follow first?

Reply:
Do you mean the rule we formatting codes ?

If so, 
yes, the plan would be using Prettier as the formatter.

There is a config file (.prettierrc.json) I proposed in [another PR which adds Prettier](https://github.com/twreporter/keystone/pull/181/files). I used the same config to format files in this change.

The import statement automatically converted to multi-line is a default behavior by using Prettier ([Prettier.printWidth](https://prettier.io/docs/en/options.html#print-width)).

Reply:
I don't have a strong opinion here. However, to avoid inconsistent style, we need to either use an automatic style fixer (e.g liner --fix) or have clear style instructions. One more question, so the prettier should be used along with eslint or jshint?

Reply:
In our case, Prettier should be used along with eslint.
We can use `eslint --fix` along with `prettier` since their functionalities complements each others while have some similarities. 

To avoid inconsistent style, I propose to use `eslint-config-prettier` to resolves the inconsistency in https://github.com/twreporter/keystone/pull/181/files.

I decided to adopt prettier as the formatter since `eslint --fix` can't fix the indent issue (mix spaces and tabs) properly.
However, there's a chance that `Eslint fix` issue has been fixes in Eslint recent releases (I am not sure).

But yes, it seems we need more explanation about why do we add `prettier` since we already have `eslint --fix`.

If the formatted changes are correct, I think we can merge it first and move this discussion to https://github.com/twreporter/keystone/pull/181/files.

reference: https://github.com/prettier/prettier-eslint/issues/101